packages/components/filters/src/filters.stories.tsx

```diff
@@ -1,20 +1,34 @@
+import Filters, { type TFiltersProps } from './filters';
 import type { Meta, StoryFn } from '@storybook/react';
-import Filters from './filters';
+import { designTokens } from '@commercetools-uikit/design-system';
 
 const meta: Meta<typeof Filters> = {
   title: 'components/Filters',
   component: Filters,
   tags: ['local-dev'],
```

Comment:
would you mind commenting this out for now so that the storybook is visible in the preview deploy for review?